### PR TITLE
XRDDEV-1773 security server views

### PR DESCRIPTION
**Pull request by Riippi**
https://jira.niis.org/browse/XRDDEV-1773

### CODE REVIEW

**Review by carohauta — CHANGES_REQUESTED** [review]

File: src/centralserver/admin-ui/src/views/SecurityServers/SecurityServersList.vue [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+   THE SOFTWARE.
+ -->
+<template>
+  <sub-view-container>
```
[the remark this answers is not in the document]
Comment (Riippi) [reply src/centralserver/admin-ui/src/views/SecurityServers/SecurityServersList.vue:27]: Fixed

File: src/centralserver/admin-ui/src/views/SecurityServers/SecurityServer/SecurityServerClients.vue [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+   THE SOFTWARE.
+ -->
+<template>
+  <div data-test="security-server-clients-view">
+    <div id="clients-filter">
+      <xrd-search v-model="search" />
```
[the remark this answers is not in the document]
Comment (Riippi) [reply src/centralserver/admin-ui/src/views/SecurityServers/SecurityServer/SecurityServerClients.vue:29]: This was discussed in slack and decided to add search. 
Margin is now fixed.

File: src/centralserver/admin-ui/src/views/SecurityServers/SecurityServer/SecurityServer.vue [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+      </router-link>
+    </div>
+    <div class="header-row">
+      <div class="title-search">
+        <div class="xrd-view-title">FOO 1</div>
+      </div>
+      <xrd-button data-test="remove-member-button"
+        ><v-icon class="xrd-large-button-icon">mdi-close-circle</v-icon>
+        {{ $t('members.deleteMember') }}</xrd-button
+      >
```
[the remark this answers is not in the document]
Comment (Riippi) [reply src/centralserver/admin-ui/src/views/SecurityServers/SecurityServer/SecurityServer.vue:41]: Fixed

**Review by carohauta — APPROVED** [review]
🦄

**Pull request merged**